pykeybasebot/chat_client.py

```diff
@@ -7,6 +8,31 @@ class ChatClient:
     def __init__(self, bot):
         self.bot = bot
 
+    async def list(self) -> List[chat1.ConvSummary]:
+        """
+        Lists your chats, with info on which ones have unread messages.
+        """
+        await self.bot.ensure_initialized()
+        res = await self.execute({"method": "list"})
+        chat_list = chat1.ChatList.from_dict(res)
+        return chat_list.conversations or []
+
+    async def read(self, channel: chat1.ChatChannel) -> List[chat1.MsgSummary]:
+        """
+        Reads the messages in a channel.
+        """
+        await self.bot.ensure_initialized()
+        res = await self.execute(
+            {"method": "read", "params": {"options": {"channel": channel.to_dict()}}}
+        )
+        thread = chat1.Thread.from_dict(res)
+        if thread.messages is None:
+            return []
+        messages = []
+        for message in thread.messages:
+            messages.append(message.msg)
```

Comment:
this looks like `go` written in python :)
how about:
```python
return [message.msg for message in (thread.messages or [])]
```
or as two lines. whatevs.